### PR TITLE
add kerberos for druid

### PR DESCRIPTION
**Pull request by pokyu**
add kerberos configure for Druid

### CODE REVIEW

**Comment by pokyu** [comment]
what can`t be done ?

**Comment by pokyu** [comment]
？

File: superset/connectors/druid/models.py
Comment (craig-rueda) [inline superset/connectors/druid/models.py]: Not sure I feel comfortable shelling out like this, from within a model, no less. This sort of logic belongs in a higher layer (connection manager, perhaps?)

File: superset/connectors/druid/models.py
Comment (craig-rueda) [inline superset/connectors/druid/models.py]: extra space?

File: requirements.txt
Comment (craig-rueda) [inline requirements.txt]: This should be in extras

**Review by craig-rueda — CHANGES_REQUESTED** [review]
Needs a few tests around Kerberos stuff. Ideally, the fetching of datasources would be broken out of the "model" and would live in another layer.

**Comment by elukey** [comment]
Any news on this pull request?

**Comment by craig-rueda** [comment]
Looks like there's merge conflicts, and there were a few comments that weren't addressed.

File: requirements.txt
Comment (mistercrunch) [reply requirements.txt:75]: +1 let's create a kerberos group in extra_requires

**Comment by mistercrunch** [comment]
FYI, the Druid connector is getting deprecated [slowly over time] in favor of using SQLAlchemy and the SQL interface to Druid. We can still merge this once the comments are addressed, but the core committers will fade out of supporting the native Druid connector. This aligns with the Druid roadmap where they're clearly doubling down on the SQL interface

**Pull request closed without merging**
